src/app/routes/utils/processRecentEpisodes/index.jsx

```diff
@@ -0,0 +1,71 @@
+import pathOr from 'ramda/src/pathOr';
+import is from 'ramda/src/is';
+
+const validateEpisode = episode => {
+  const checks = [
+    episode,
+    is(String, episode.id),
+    episode.id.includes(':'),
+    episode.brand,
+    is(String, episode.brand.title),
+    episode.media,
+    is(String, episode.media.id),
+    is(String, episode.media.title),
+    is(Array, episode.media.versions),
+    episode.media.versions[0],
+    is(String, episode.media.versions[0].durationISO8601),
+    is(Number, episode.media.versions[0].availableFrom),
+  ];
+  // TODO: log if invalid
+
+  return checks.every(Boolean);
+};
+
+// We have different requirements depending on whether or not editorial title their episodes
+// However, the data always contains a title - ARES populate a default one if it is not defined
+// This function checks if the title appears to be that fallback title, and removes it if so
+const formatTitle = title => {
```

Comment:
Don't think this is necessary - there are two values: episodeTitle and title

episodeTitle is the one entered by editorial
title is the default date